crates/tests/databases-tests/src/postgres/configuration_tests.rs

```diff
@@ -15,7 +15,8 @@ use tests_common::common_tests;
 
 #[tokio::test]
 async fn get_configuration_schema() {
-    let schema = schemars::schema_for!(ndc_postgres_configuration::RawConfiguration);
```

Comment:
Schemas used to contain the union of all supported versions. Now they relate only to a single version (like they used to when there was only a single version)